### DISCUSSION TITLE
RecordedFutures YARA files

### DISCUSSION THREAD

**Discussion by jpancrazio** [Security-Onion-Solutions/securityonion#13405] [Unsupported Versions]
Hello All, I have searched the docs ( not asking about 3rd party intigration of RF, Just want to add the supplied YARA files from RF, But I am unsure totally how to add them . I see the options for additional repos etc . but I simply have the yml and yara files - Any help would be appreciated

**Comment by reyesj2** [comment]
You can look at adding them via the detections UI, but if its alot of rules and you already have the .yar rule files. The best option is going to be creating a repo with those rules. 

You can create a new directory and do ```git init``` to start a new repo.  Create a new directory in your repo like 'yara' and add the .yar files to the /yara/ directory in your git repo.

That git repo can get copied over to /nsm/rules/ on your Security Onion Manager node.

Then add the configuration for this new repo to SOC as shown https://docs.securityonion.net/en/2.4/yara.html#custom-yara-repositories
```{"community":true,"folder":"yara","license":"Elastic-2.0","repo":"file:///nsm/rules/new-git-repo"}``` 

In Detections you can run a full sync for Strelka and you should see the new rules imported
